android/kotlin/AmpliApp/README.md

```diff
@@ -0,0 +1,30 @@
+# Ampli Android SDK (Kotlin)
+An example app using the Ampli Android SDK
+
+# Usage
+
+### Setup the project
+You will need to do the following before running the app.
+1. Create new file `src/main/res/values/secrets.xml` with your Amplitude API key:
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="AMPLITUDE_API_KEY">your-amplitide-api-key</string>
+</resources>
+```
+
```

Comment:
Can you please add a `local.properties.example` with `#sdk.dir=/path/to/Android/sdk` and reference it here

Reply:
Done. Not sure how to write text correctly